recipe/meta.yaml

```diff
@@ -30,8 +33,17 @@ requirements:
 
 test:
   commands:
+    # Check if NAD27/NAD83 datum shifting is present if `conus` is present.
+    - test -f $PREFIX/share/proj/conus  # [not win]
+    - if not exist %PREFIX%\\Library\\SHARE\\conus exit 1  # [win]
+    # In the env these are automatically set but we need them here for testing.
+    - set PROJ_LIB=%PREFIX%\\Library\\SHARE  # [win]
+    - export PROJ_LIB=$PREFIX/share/proj  # [linux]
```

Comment:
That is awesome @msarahan! Conda-forge really needs to start using `conda-build 2`.